### PR TITLE
Style form builder and rendered forms to match bc gov look/feel

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

supports [work on styling the form ](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1608)

We need the form builder and the forms it creates to have the look and feel of the bcgov style guidelines.

not yet sure if there is a reasonable way to modify html markup produced by the form builder. 
so we can apply css/scss rules (similar to those in vuetify) to the formio markup.
The SASS @extend directive can be useful here.

long term approach could be:
- modify the html markeup produced by the formio form builder, so that each component has html elements (tags) and class names that can be styled using vuetify and our base stylesheet.

example:
bcgov prefers the 'description' text for a text input to appear above the field, but formio show it under the field.
formio use 'text-muted' to make it appear grey, but there is now class to make it smaller than default text (12px).
it would be nice to move the markup around and add a 'small' class. i think this can be done by modifying our custom simpletextfiled component. 

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-48

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-48

File: app/frontend/src/assets/scss/style.scss [hunk 1]
```diff
@@ -2,14 +2,23 @@
  * BCGov Style
  * This globally extends the base Vuetify style
  */
-@import '~vuetify/src/styles/styles.sass';
+ @import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900');
+ @import url('https://fonts.googleapis.com/css?family=Material+Icons');
+ @import "~material-design-icons-iconfont/dist/material-design-icons.css";
+ @import "~font-awesome/css/font-awesome.css";
+ @import '~vuetify/src/styles/styles.sass';
```
Comment (jujaga) [inline app/frontend/src/assets/scss/style.scss:9]: This is over-indented by 1 space.

File: app/frontend/src/assets/scss/style.scss [hunk 2]
```diff
@@ -42,6 +51,14 @@ p {
   line-height: 1.4;
 }
 
+// links
+a{
```
Comment (jujaga) [inline app/frontend/src/assets/scss/style.scss:55]: Make sure to use VSCode autoformatter to keep spacing consistent. I think there's supposed to be a space between a and { but can't remember SCSS conventions.

File: app/frontend/src/components/designer/FormViewer.vue [hunk 3]
```diff
@@ -220,3 +220,10 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+// include bootstrap and formio styles for the form builder
+@import '~bootstrap/dist/css/bootstrap.min.css';
+@import 'https://unpkg.com/formiojs@4.11.2/dist/formio.builder.min.css';
+</style>
+
```
Comment (jujaga) [inline app/frontend/src/components/designer/FormViewer.vue:229]: Extra whitespace

File: app/frontend/src/assets/scss/style.scss [hunk 4]
```diff
@@ -114,3 +131,112 @@ a, .v-tab {
...
+    border-radius: 3px;
+    box-shadow: 0 2px 8px 0 rgba(175, 175, 175, 0.5);
+    font-size: .85em;
+  }
+
+  // validation messages and help text
+  .form-text{
+    font-size: .85em;
+  }
+  .invalid-feedback{
```
Comment (jujaga) [inline app/frontend/src/assets/scss/style.scss:200]: Make sure to use VSCode autoformatter to keep spacing consistent. I think there's supposed to be a space between feedback and { but can't remember SCSS conventions.

**Review by jujaga — COMMENTED** [review]
Looks pretty good from what I can tell. Please take a look at the minor code styling/format comments when you can.

**Comment by TimCsaky** [comment]
I've just imported the Kitchen sink form schema that includes all the components to get a better idea of what needs styling. and it looks like there are still a few more improvements I can make. I guess this is a work in progress. but i'll make a few more tweaks to this pr now

File: app/frontend/src/assets/scss/style.scss [hunk 5]
```diff
@@ -2,14 +2,23 @@
  * BCGov Style
  * This globally extends the base Vuetify style
  */
-@import '~vuetify/src/styles/styles.sass';
+ @import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900');
+ @import url('https://fonts.googleapis.com/css?family=Material+Icons');
+ @import "~material-design-icons-iconfont/dist/material-design-icons.css";
+ @import "~font-awesome/css/font-awesome.css";
```
Comment (loneil) [inline app/frontend/src/assets/scss/style.scss:8]: I don't think we need FA on the global scope, it's only used by the bootstrap controls isn't it?

Comment (TimCsaky) [reply app/frontend/src/assets/scss/style.scss]: that would be nice. i think you're right. i'll try moving this import to the component.

File: app/frontend/src/assets/scss/style.scss [hunk 6]
```diff
@@ -103,14 +135,200 @@ a, .v-tab {
...
+  // tabs
+  .formio-component-simpletabs {
+    & > .card {
+      border: 0;
+
+      & > .card-body {
+        border-style: solid;
+        border-color: rgba(0, 0, 0, 0.125);
+        border-width: 0 1px 1px 1px;
+        border-radius: 0 0px 0 0px;
```
Comment (loneil) [inline app/frontend/src/assets/scss/style.scss:188]: This can just be `border-radius: 0`

**Review by loneil — APPROVED** [review]
Looking good

**Review by jujaga — APPROVED** [review]

**Pull request merged**